README.md

```diff
@@ -1,6 +1,9 @@
 # go-google-scraper-challenge
 A project for Nimble Go Internal Certification on Web
 
+[Staging](https://google-scraper-staging.herokuapp.com)
+[Production](https://google-scraper-web.herokuapp.com/)
```

Comment:
Can we remove `/` from the end of the url? or we can add `/` for production url also for consistency